Client Development/Client Scripts/Illumio Max Retry Count onChange.js

```diff
@@ -0,0 +1,10 @@
+function onChange(control, oldValue, newValue, isLoading, isTemplate) {
+    if (isLoading || newValue === '') {
+        return;
+    }
+	newValue = newValue.replaceAll(',','');
+    var val = parseInt(newValue, 10);
+    if (val < 0 || val > 100) {
```

Comment:
Can we use the constants here and in the messages below?

Reply:
The constants are in the Server side script include, and this is client side code, due to which we cannot have it here

Reply:
No, we cannot access the gs in the client side

Reply:
So the only options are scratchpad or GlideAjax? Can we prepopulate the scratchpad with these values? Having these constants in multiple places makes it difficult if we decide to change them later.

Reply:
If we are gonna use the UI script, i can try to see if the constants are directly accessible there or not, as i have seen some server side APIs in it in the global scope

Reply:
Cool, that would simplify things